vhost-device-vsock/src/vhu_vsock.rs

```diff
@@ -142,13 +148,27 @@ impl std::convert::From<Error> for std::io::Error {
     }
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub(crate) struct VsockProxyInfo {
+    pub forward_cid: u32,
+    pub listen_ports: Vec<u32>,
+}
+
+#[derive(Debug, PartialEq, Clone)]
+pub(crate) enum ProxyType {
```

Comment:
I'd call it something like `BackendType`

Reply:
Great suggestion. Done.